### PR TITLE
[pt-br] Update pt-br/community/_index.html

### LINKED ISSUE #27446: [pt-br] Update the community pages to reflect the english version / Atualizar a página de comunidades para refletir a versão em Inglês
Community pages in portuguese is outdated. We need to update it to reflect the layout and the text from english version

A página de comunidades (content/pt-br/community) está desatualizada, em relação à página em Português, faltando imagens, o layout adequado, etc.

/good-first-issue
/help
/language pt

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes
* Update the communities page to reflect the latest English version.

Fixes #27446

### CODE REVIEW

File: content/pt-br/community/_index.html [hunk 1]
```diff
@@ -2,70 +2,198 @@
...
+      <a href="https://github.com/kubernetes/kubernetes">GitHub&nbsp;&#9654;</a>
+      <p>Todo o controle do projeto e de <i>issues</i>, além do código, logicamente.</p>
+    </div>
+
+    <div id="server-fault" class="community-resource">
+      <a href="https://serverfault.com/questions/tagged/kubernetes">
+        <img src="/images/community/serverfault.png" alt="Server Fault">
+      </a>
+      <a href="https://serverfault.com/questions/tagged/kubernetes">Server Fault&nbsp;&#9654;</a>
+      <p>Discussões relactionadas ao Kubernetes no site Server Fault. Faça uma
```
Comment (MrErlison) [inline content/pt-br/community/_index.html:160]: typo: relactionadas

**Comment by MrErlison** [comment]
Hello @stormqueen1990! Thanks for your contribution!

File: content/pt-br/community/_index.html [hunk 2]
```diff
@@ -2,70 +2,198 @@
...
+      <iframe src="https://www.youtube.com/embed/videoseries?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg" title="Monthly office hours" allow="camera 'none'; microphone 'none'; geolocation 'none'; fullscreen https://www.youtube.com/" ></iframe>
+      <a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg">
+       <span class="videocta">Assista às reuniões de "office hours" mensais&nbsp;&#9654;</span>
+      </a>
+    </div>
+
+    <div class="video youtube">
+      <iframe src="https://www.youtube.com/embed/videoseries?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ" title="Weekly community meetings" allow="camera 'none'; microphone 'none'; geolocation 'none'; fullscreen https://www.youtube.com/"></iframe>
+      <a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ">
+        <span class="videocta">Assista às reuniões da comunidade semanais&nbsp;&#9654;</span>
```
Comment (MrErlison) [inline content/pt-br/community/_index.html:109]: Sounds better to me "Assista às reuniões semanais da comunidade". Think about it.

File: content/pt-br/community/_index.html [hunk 3]
```diff
@@ -2,70 +2,198 @@
...
+  <p class="community-simple">
+    O Kubernetes aparece bastante no YouTube. Inscreva-se para ver mais sobre
+    vários tópicos.
+  </p>
+
+  <div class="container">
+    <div class="video youtube">
+      <iframe src="https://www.youtube.com/embed/videoseries?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg" title="Monthly office hours" allow="camera 'none'; microphone 'none'; geolocation 'none'; fullscreen https://www.youtube.com/" ></iframe>
+      <a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg">
+       <span class="videocta">Assista às reuniões de "office hours" mensais&nbsp;&#9654;</span>
```
Comment (MrErlison) [inline content/pt-br/community/_index.html:102]: Sounds better to me 'Assista às reuniões mensais de "office hours"'. Think about it.

File: content/pt-br/community/_index.html (hunk 2 above)
Comment (stormqueen1990) [reply content/pt-br/community/_index.html]: Updated!

File: content/pt-br/community/_index.html (hunk 3 above)
Comment (stormqueen1990) [reply content/pt-br/community/_index.html]: Updated!

File: content/pt-br/community/_index.html (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/community/_index.html]: Updated!

**Comment by MrErlison** [comment]
Beautiful work @stormqueen1990. I really liked this page in Portuguese.
/lgtm

File: content/pt-br/community/_index.html [hunk 4]
```diff
@@ -2,70 +2,198 @@
...
-            </div>
-            <div>
-                <a href="http://stackoverflow.com/questions/tagged/kubernetes">Stack Overflow</a>
-                <p>Nosso fórum de usuários é um ótimo lugar para o suporte da comunidade.</p>
-            </div>
-        </div>
-    </main>
-</section>
+<div class="community-section" id="introduction">
+  <p>A comunidade do Kubernetes - usuários, contribuidores, e a culura que nós
```
Comment (edsoncelio) [inline content/pt-br/community/_index.html:15]: typo: cultura

Comment (stormqueen1990) [reply content/pt-br/community/_index.html]: Fixed!

**Comment by edsoncelio** [comment]
Thank you!!
/lgtm

**Pull request merged**